internal/cli/client/client.go

```diff
@@ -0,0 +1,33 @@
+package client
+
+import (
+	"context"
+	"fmt"
+	"octavius/pkg/protobuf"
+	"time"
+)
+
+type Client interface {
+	CreateJob(*protobuf.RequestForMetadataPost) error
+}
+
+type grpcClient struct {
+	client                protobuf.OctaviusServicesClient
+	connectionTimeoutSecs time.Duration
+}
+
+func NewGrpcClient(client protobuf.OctaviusServicesClient) Client {
+	return &grpcClient{
+		client:                client,
+		connectionTimeoutSecs: time.Second,
+	}
+}
+
+func (g *grpcClient) CreateJob(metadataPostRequest *protobuf.RequestForMetadataPost) error {
+	res, err := g.client.CreateJob(context.Background(), metadataPostRequest)
+	if err != nil {
+		return err
+	}
+	fmt.Println(res)
```

Comment:
instead of fmt.Println better if we use logger methods. This will be a good practice to implement from start. I will mark all the places where logger is needed